src/Makefile.am

```diff
@@ -1248,10 +1248,39 @@ AM_CPPFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/include/tdep-$(arch) -I.
 AM_CCASFLAGS = $(AM_CPPFLAGS)
 noinst_HEADERS += unwind/unwind-internal.h
 
-EXTRA_DIST = $(libunwind_la_EXTRAS_ia64)
+EXTRA_DIST = $(libunwind_la_EXTRAS_ia64) abi
 
 MAINTAINERCLEANFILES = Makefile.in
 
+#
+# ABI checks
+#
+# Running the ABI checks require configuring with CFLAGS="-g -Og" and running
+# the targets explicitly with `make -C src abi-check`. The result is output in a
+# file with the `.abidiff` extension.
+#
+# This also requires libabigail tools of at least version 2.0 to be installed.
```

Comment:
Ubunt 20.04 TLS  still includes 1.6. I think that justifies a configure check, which then also includes the `--enable-shared` check

Reply:
Done.